cypress/support/bodyTestHelper.js

```diff
@@ -21,6 +21,70 @@ export const checkElementStyles = (elementString, text, color, fontFamily) => {
   shouldContainStyles(el, 'font-family', fontFamily);
 };
 
+export const getBlockData = (blockType, win) => {
+  let blockData;
+  const { blocks } = win.SIMORGH_DATA.data.content.model;
+
+  blocks.forEach(block => {
+    if (!blockData && block.type === blockType) {
+      blockData = block;
+    }
+  });
+  return blockData;
+};
+
+export const firstHeadlineDataWindow = () => {
+  cy.window().then(win => {
+    const headlineData = getBlockData('headline', win);
+    const { text } = headlineData.model.blocks[0].model.blocks[0].model;
+    checkElementStyles(
+      'h1',
+      text,
+      'rgb(34, 34, 34)',
+      'ReithSerifNewsMedium, Helvetica, Arial, sans-serif',
+    );
+  });
+};
+
+export const firstSubheadlineDataWindow = () => {
+  cy.window().then(win => {
+    const subheadingData = getBlockData('subheadline', win);
+    const { text } = subheadingData.model.blocks[0].model.blocks[0].model;
```

Comment:
My instinct would be to make another function which handles the `.model.blocks[0].model.blocks[0].model` (can then refactor lines 52 and 39), but as the business logic is so specific to each component, I wouldn't bother making a method _just_ for that. For instance, line 76 is `.model.blocks[0].model`. 

But what we could do is write a _generic_ method 🤔 something like:

```js
    const { text } = getModelThisManyLayersDeep(subheadingData, 3);
```

(Line 76 would then be `getModelThisManyLayersDeep(copyrightData, 2)`)

Definition would be something like:

```js
export const getModelThisManyLayersDeep = (data, layers) => {
  if (!layers || layers === 1) {
    return data.model;
  }
  else {
    let { model } = data;
    while (layers > 0) {
        model = data.blocks[0].model;
        layers--;
    }
    return model;
}
```

Reply:
^ We should do this in our application code to make it more readable too 😀